test/unit/presenters/publishing_api/fatality_notice_presenter_test.rb

```diff
@@ -67,14 +69,14 @@ class PublishingApi::FatalityNoticePresenterTest < ActiveSupport::TestCase
     expected_links = {
       field_of_operation: [@fatality_notice.operational_field.content_id],
       ministers: [],
-      organisations: [],
+      organisations: %w[c377bbc7-94f6-4d81-8636-295e72df9f52],
       people: [],
-      policy_areas: [],
-      primary_publishing_organisation: [],
-      original_primary_publishing_organisation: [],
+      policy_areas: %w[e21e9b56-8a77-4fcf-b183-41f508e40a63],
+      primary_publishing_organisation: %w[c377bbc7-94f6-4d81-8636-295e72df9f52],
+      original_primary_publishing_organisation: %w[c377bbc7-94f6-4d81-8636-295e72df9f52],
```

Comment:
Are this random? What do they mean?

Reply:
These are actual ID's that are part of the content. The test checks that edition links exists so I have updated it.

Reply:
Is it possible to have them like `policy_areas: %w[@some_policy_area.id],` or something similar, instead of actual ids? It would make the test clearer.

Reply:
@AlanGabbianelli the tests are passing without the ID's, so no changes needed for the test and PR is upgrading factory_bot so looks like all is well 👍